readme.md

```diff
@@ -170,7 +227,7 @@ numbers = [ 1, 2, 3, 4, 5 ]
 numbers.each { |number| puts number * 2 }
 ```
 
-**DIAGRAM:** EACH
+**Visualize:** EACH: using this [code visualizer](http://pythontutor.com/ruby.html#mode=edit) let's look at how `each` operates under the hood
```

Comment:
I feel like it might be good to have a whiteboard version of this ready too, since it might be a bit simpler and more clear than just the code viz tool (which you should definitely keep here too)